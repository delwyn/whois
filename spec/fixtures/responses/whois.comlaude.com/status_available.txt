-------------------------------------------------------------------------------
Com Laude registers, maintains and renews domain names around the world for
leading intellectual property owners and the law firms that support them.
If you have queries about this domain, you may contact us via our website
at www.comlaude.com.
-------------------------------------------------------------------------------

The data in the Com Laude Whois database is provided to assist you in obtaining
information about a domain name registration record.  Com Laude makes this
information available "as is," and does not guarantee its accuracy.
By submitting a WHOIS query, you agree that you will use this data only for
lawful purposes and that, under no circumstances will you use this data to:

(1) allow, enable, or otherwise support the transmission of mass unsolicited,
commercial advertising or solicitations via e-mail (spam), telephone or fax; or
(2) enable high volume, automated, electronic processes that that send queries
or data to Com Laude or the systems of any Registry Operator or ICANN-Accredited
registrar.  The compilation, repackaging, dissemination or other use of this
data is expressly prohibited without the prior written consent of Com Laude.

Com Laude reserves the right to modify these terms at any time. By submitting
this query, you agree to abide by these terms. If you fail to abide by this
policy, we may terminate your access to this WHOIS database.


No match for "notmine.com".



-------------------------------------------------------------------------------
Com Laude protects intellectual property in the domain name system. Com Laude
only registers domain names for legitimate rights owners. Should you have any
queries about the legitimacy of this name or our work as a UK based, ICANN
Accredited Registrar, contact us at: Com Laude, 116 Long Acre, Covent Garden,
London WC2E 9SU, UK.  Com Laude is a business name of Nom IQ Ltd.
-------------------------------------------------------------------------------
