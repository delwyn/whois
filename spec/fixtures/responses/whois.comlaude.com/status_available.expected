#status
  should: %s raise_error(Whois::AttributeNotSupported)

#available?
  should: %s == true

#registered?
  should: %s == false


#created_on
  should: %s == nil

#updated_on
  should: %s raise_error(Whois::AttributeNotSupported)

#expires_on
  should: %s == nil


#registrar
  should: %s == nil


#registrant_contacts
  should: %s CLASS(array)
  should: %s SIZE(0)

#admin_contacts
  should: %s CLASS(array)
  should: %s SIZE(0)

#technical_contacts
  should: %s CLASS(array)
  should: %s SIZE(0)


#nameservers
  should: %s CLASS(array)
  should: %s SIZE(0)
